Sources/Db/APIs/PostgreSQL.php

```diff
@@ -2117,6 +2117,11 @@ protected function __construct(array $options = [])
 			self::$db_connection = $this->connection;
 		}
 
+		// At this point, if we don't have a connection, nothing else can be done.
+		if (empty($this->connection)) {
+			return;
+		}
+
```

Comment:
As with MySQL.php, this should go before the attempt to set `self::$db_connection`.